handbook/.pages

```diff
@@ -4,4 +4,5 @@ nav:
     - rbac
     - openid-connect
     - ldap
+    - data-stores
```

Comment:
Unfortunately we don't have a lot of control of this behavior for now due to how the awesome-pages plugin works. Since I've removed the File storage placeholder document it doesn't exhibit this odd behavior.